### PR TITLE
Fixed #12651: providing links to the  .expanding in .cum*

### LINKED ISSUE #12651: DOC: improve cross-links in docs between .expanding and .cum*
xref #12648 
- cross-link doc-strings for .cum\* to the .expanding().\* (for sum,prod,max,min, which are only supported).
- show an example (in computation.rst wher eexpanding is shown? or basics.rst (where cum\* are shown)) how these are related (w.r.t. NaN filling).

```
In [7]: s = Series([1,2,np.nan,3,np.nan,4])

In [8]: s.cumsum()
Out[8]: 
0     1.0
1     3.0
2     NaN
3     6.0
4     NaN
5    10.0
dtype: float64

In [9]: s.expanding().sum()
Out[9]: 
0     1.0
1     3.0
2     3.0
3     6.0
4     6.0
5    10.0
dtype: float64

In [10]: s.cumsum().ffill()
Out[10]: 
0     1.0
1     3.0
2     3.0
3     6.0
4     6.0
5    10.0
dtype: float64
```

### PR DESCRIPTION
**Pull request by prabhjotsumman**
- [ x] closes #12561
- [ ] tests added / passed
- [ ] passes `git diff upstream/master | flake8 --diff`
- [x] whatsnew entry
#12561 : wrote the `.expanding` function int the `basics.rst`

### CODE REVIEW

**Comment by sinhrks** [comment]
xref #12673. Should close #12651 (not #12561).

File: doc/source/basics.rst [hunk 1]
```diff
@@ -466,6 +466,12 @@ preserve the location of NA values:
 
    df.cumsum()
 
+Also, refer to the ``expanding()`` function which is similar to the ``cumsum`` function.
+
+.. ipython:: python
+
+   df.expanding()
```
Comment (sinhrks) [inline doc/source/basics.rst]: Should be ``.expanding().sum()``, same as #12673 (link via docstring, use section link, or add explicit explanation).

**Comment by Prabhnith** [comment]
sectional for the expanding provided.

**Comment by jreback** [comment]
move these commits to #12673

**Pull request closed without merging**